app/views/users/profile.html.erb

```diff
@@ -72,7 +72,7 @@
   </div>
 
   <hr>
-  
+
```

Comment:
Quick tip: Remove unwanted lines of code from the PR before code reviews. This helps in error free quick pr reviews.
Thanks